test/test.config.js

```diff
@@ -24,3 +26,29 @@ config.mongodb.dropCollections.collections = [];
 
 // allow self-signed certs in test framework
 config['https-agent'].rejectUnauthorized = false;
+
+// this needs to be true in order to set auth stuff
+config['account-http'].autoLoginNewAccounts = true;
+
+config.express.session.secret = 'NOTASECRET';
+config.express.session.key = 'web-authn-token-test-session';
+config.express.session.prefix = 'web-authn-token-test';
+// test specific settings
+config.express.session.rolling = true;
+config.express.session.resave = true;
+config.express.session.saveUninitialized = false;
+
+// FIXME change this to a common setting
```

Comment:
What does this mean?  Can it be resolved?

Reply:
this is being resolved in the session expiration work that this test suite will help with. So basically in the PR for session expire this FIXME should be removed. This is just a reminder to do it then.

Reply:
Please remove this then as it's not actionable by anyone but yourself and create a github issue if you need a reminder to do something.  Generally speaking, adding TODO's FIXME's to code going forward is not really going to be a thing.  It's too difficult to identify and triage FIXMEs and TODOs months after the fact.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-web-session/pull/9/commits/68a177b0cbcb2b52ac7eb1f27fe6acbb50cf8f26